src/components/input/Select.tsx

```diff
@@ -478,6 +484,7 @@ function SelectMain<T>({
   containerClasses,
   onListboxScroll,
   right = false,
+  alignListbox = right ? 'right' : 'left',
```

Comment:
For now we define the default value of `alignListbox` based on `right`, but as soon as we remove `right` in the next major version, we can change this to `alignListbox = 'left'`.